pkg/operator/mon/pod.go

```diff
@@ -18,37 +18,47 @@ package mon
 import (
 	"fmt"
 
-	"github.com/rook/rook/pkg/cephmgr/mon"
 	"github.com/rook/rook/pkg/operator/k8sutil"
-	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/pkg/api/v1"
 	"k8s.io/client-go/pkg/labels"
 )
 
+func ClusterNameEnvVar() v1.EnvVar {
```

Comment:
this config map/env var approach is going to come in handy for other pod specs i have a feeling...